testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AvailableJavaHomes.java

```diff
@@ -122,60 +124,112 @@ public static Jvm getJdk24() {
         return getJdk(JavaVersion.VERSION_24);
     }
 
-    @Nullable
-    public static Jvm getHighestSupportedLTS() {
-        return getJdk21();
+    /**
+     * Get a JDK for each major Java version installed on this machine.
+     */
+    public static List<Jvm> getAllJdkVersions() {
+        return getJdksInRange(Range.atLeast(0));
```

Comment:
💭 `Range.all` exists, thought it doesn't materially matter.
```suggestion
        return getJdksInRange(Range.all());
```

Reply:
Already triggered this for merge. Will include this in the next set of changes